main/main.c

```diff
@@ -45,6 +46,59 @@ uint64_t cycles_per_sec;
 uint64_t cycles_per_ms;
 uint64_t picosec_per_cycle;
 
+char *lua_base_dir = NULL;
+char *gatekeeper_config_file = NULL;
+
+/* Argp's global variables. */
+const char *argp_program_version = "gatekeeper 1.0";
+
+/* Arguments. */
+static char adoc[] = "<LUA_BASE_DIR> <GATEKEEPER_CONFIG_FILE>";
```

Comment:
Variable `adoc` should be `""` (i.e. the empty string) because Gatekeeper requires no arguments. Gatekeeper has options, though.